snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/core/domain/SnomedConcept.java

```diff
@@ -342,6 +371,7 @@ public SnomedConcepts getStatedDescendants() {
 		return statedParentIds == null ? null : Arrays.stream(statedParentIds).mapToObj(Long::toString).collect(Collectors.toList());
 	}
 	
+	@JsonIgnore
```

Comment:
I'm afraid of situations where someone sends in JSON with an updated enumeration value, and we don't act on it 😨 

Can/should we keep writeability while it is present in the JSON output?

Reply:
We use special input types on the REST API side, that will prevent the scenario you are describing.